src/LibraryManager/Manifest.cs

```diff
@@ -109,7 +109,6 @@ internal static Manifest FromJson(string json, IDependencies dependencies)
             }
             catch (Exception)
             {
-                dependencies.GetHostInteractions().Logger.Log(PredefinedErrors.ManifestMalformed().Message, LogLevel.Task);
                 return null;
```

Comment:
Prafull, can you please make sure the CLI is logging errors for manifests that fails to parse. GetManifestErrorsAsync returns the proper error for Null manifests.